### DISCUSSION TITLE
Title updated if in template?

### DISCUSSION THREAD

**Discussion by superhu4** [notaapp/talk#252] [Questions]
Note names are usually updated as long as "Auto-rename notes to title" is selected.

However, I've noted that if I use a template, the name is permanently set, and changing the title does not change the name of the note.

Is this the expected behavior?

**Comment by astoilkov** [comment]
Currently, if the `{{title}}` isn't placed inside of the H1 it won't update. We are considering updating the title regardless of its position.

Where have you placed your `{{title}}`?

**Comment by superhu4** [comment]
Thank you.

I had the `{{title}}` tag inside the H1, but it was prepended with `{{date}} - `

Thus:

`{{date}} - {{title}}`

I think the `{{date}} - ` before the `{{title}}` prevented the title from updating.

**Reply by astoilkov** [reply]
Got it.

You are right. The `{{date}} - ` was preventing the title from updating. We will try to think of a solution for this. Thanks!

**Comment by superhu4** [comment]
Thank you.

I had the {{title}} tag inside the H1, but it was prepended with {{date}} -

Thus:

{{date}} - {{title}}

I think the {{date}} - before the {{title} prevented the title from
updating.
